### ISSUE TITLE
Can anyone get the results of scannet in paper?

### ISSUE THREAD

**Issue by LinZhuoChen** [MVIG-SJTU/pointSIFT#19]
I have not changed the code of pointsift and I can't get the result reported in paper,The result in paper is 86.1, I can only get 84.8, Can someone get the results of scannet in paper?Maybe I made some mistake,Thank you!

**Comment by sjtuytc** [comment]
Yes, and after some modification I can get 86.3.
You can check in your tensorboard.

**Comment by meteorshowers** [comment]
@sjtuytc   can you describe your modification? And, how many GPU you use to train the model to get the 86.3? Thanks a lot!

**Comment by meteorshowers** [comment]
@sjtuytc  my Wechat is 16600286930, thanks a lot!

**Comment by WuzhaoLee** [comment]
@sjtuytc  Could you please describe your modification? I both use 2 or 4 GPU but I can only get 85.42.

**Comment by sjtuytc** [comment]
I add more shortcuts, and the modified network model is with the attachment.

----- 原始邮件 -----
发件人: "WuzhaoLi" <notifications@github.com>
收件人: "MVIG-SJTU/pointSIFT" <pointSIFT@noreply.github.com>
抄送: "sjtuytc" <sjtuytc@sjtu.edu.cn>, "Mention" <mention@noreply.github.com>
发送时间: 星期五, 2019年 1 月 11日 下午 3:12:55
主题: Re: [MVIG-SJTU/pointSIFT] Can anyone get the results of scannet in paper? (#19)

@sjtuytc  Could you please describe your modification? I both use 2 or 4 GPU but I can only get 85.42.

**Comment by WuzhaoLee** [comment]
@sjtuytc oh~ Thanks for your reply. But I can't find the attachment everywhere 😂... Could you please send the modified network model again? Thanks a lot~

**Comment by sjtuytc** [comment]
dot py isn't supported by Github
[pointSIFT_pointnet.zip](https://github.com/MVIG-SJTU/pointSIFT/files/2748596/pointSIFT_pointnet.zip)

**Comment by sjtuytc** [comment]
> @sjtuytc oh~ Thanks for your reply. But I can't find the attachment everywhere 😂... Could you please send the modified network model again? Thanks a lot~

**Comment by WuzhaoLee** [comment]
@sjtuytc Thanks！😁

**Comment by iris0329** [comment]
@sjtuytc  hi, Thanks for your sharing the code. 
but I found the loss stopped decreasing at about 11. 
Could you tell me how you set the parameters? such as how many epoches you set?

**Comment by sjtuytc** [comment]
One year has past and I could not remember those details, sorry. Only learning rate matters, I remember.

**Comment by iris0329** [comment]
@sjtuytc  thank you !

**Comment by liarba** [comment]
> I have not changed the code of pointsift and I can't get the result reported in paper,The result in paper is 86.1, I can only get 84.8, Can someone get the results of scannet in paper?Maybe I made some mistake,Thank you!

Hi，I didn't change any code and I got calibrated vox accuracy around 84.6%, for vox accuracy I can only get 83.0%.  Do you also get calibrated vox accuracy 84.8% with the original code?

**Comment by sjtuytc** [comment]
> > I have not changed the code of pointsift and I can't get the result reported in paper,The result in paper is 86.1, I can only get 84.8, Can someone get the results of scannet in paper?Maybe I made some mistake,Thank you!
> 
> Hi，I didn't change any code and I got calibrated vox accuracy around 84.6%, for vox accuracy I can only get 83.0%. Do you also get calibrated vox accuracy 84.8% with the original code?

Note the reported result is the best acc over all epoches, not the last epoch.

**Comment by liarba** [comment]
> > > I have not changed the code of pointsift and I can't get the result reported in paper,The result in paper is 86.1, I can only get 84.8, Can someone get the results of scannet in paper?Maybe I made some mistake,Thank you!
> > 
> > 
> > Hi，I didn't change any code and I got calibrated vox accuracy around 84.6%, for vox accuracy I can only get 83.0%. Do you also get calibrated vox accuracy 84.8% with the original code?
> 
> Note the reported result is the best acc over all epoches, not the last epoch.

Yes, I've noticed that, but the best vox average accuracy is still 83.0%. The best calibrated vox accuracy can reach to  84.6%. I didn't change any code and run on 4 gpus with 1000 epochs.

**Comment by sjtuytc** [comment]
> > > > I have not changed the code of pointsift and I can't get the result reported in paper,The result in paper is 86.1, I can only get 84.8, Can someone get the results of scannet in paper?Maybe I made some mistake,Thank you!
> > > 
> > > 
> > > Hi，I didn't change any code and I got calibrated vox accuracy around 84.6%, for vox accuracy I can only get 83.0%. Do you also get calibrated vox accuracy 84.8% with the original code?
> > 
> > 
> > Note the reported result is the best acc over all epoches, not the last epoch.
> 
> Yes, I've noticed that, but the best vox average accuracy is still 83.0%. The best calibrated vox accuracy can reach to 84.6%. I didn't change any code and run on 4 gpus with 1000 epochs.

Yes, you're right. You need to use my modified code.
